package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@istreamplanet/pebble",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
You do not manually do this. This is done after merge when you run the `npm version` command on the main branch.

Reply:
@osbornm nope, I didn't touch it. Maybe it was from before?